tests/functional/binary-cache.sh

```diff
@@ -25,49 +25,49 @@ for path in "${paths[@]}"; do
 done
 
 # Test copying build logs to the binary cache.
-expect 1 nix log --store file://$cacheDir $outPath 2>&1 | grep 'is not available'
-nix store copy-log --to file://$cacheDir $outPath
-nix log --store file://$cacheDir $outPath | grep FOO
-rm -rf $TEST_ROOT/var/log/nix
-expect 1 nix log $outPath 2>&1 | grep 'is not available'
-nix log --substituters file://$cacheDir $outPath | grep FOO
+expect 1 nix log --store file://"$cacheDir" "$outPath" 2>&1 | grep 'is not available'
+nix store copy-log --to file://"$cacheDir" "$outPath"
+nix log --store file://"$cacheDir" "$outPath" | grep FOO
+rm -rf "$TEST_ROOT"/var/log/nix
+expect 1 nix log "$outPath" 2>&1 | grep 'is not available'
+nix log --substituters file://"$cacheDir" "$outPath" | grep FOO
 
 # Test copying build logs from the binary cache.
-nix store copy-log --from file://$cacheDir $(nix-store -qd $outPath)^'*'
-nix log $outPath | grep FOO
+nix store copy-log --from file://"$cacheDir" "$(nix-store -qd "$outPath")"^'*'
+nix log "$outPath" | grep FOO
 
 basicDownloadTests() {
     # No uploading tests bcause upload with force HTTP doesn't work.
 
     # By default, a binary cache doesn't support "nix-env -qas", but does
     # support installation.
-    clearStore
+   clearStore
```

Comment:
indentation issue?

Reply:
Fortunately, this doesn't affect the test itself and shellcheck is still happy.

```bash
bash-5.2$ ./mk/run-test.sh tests/functional/binary-cache.sh
ran test tests/functional/binary-cache.sh... [PASS]
bash-5.2$ shellcheck tests/functional/binary-cache.sh
bash-5.2$ echo $?
0
```

I went ahead and rebased, fixed, and force pushed.